### ISSUE TITLE
Throwing error looking for character-sheet.js in v10

### ISSUE THREAD

**Issue by tehguitarist** [ElfFriend-DnD/foundryvtt-compactBeyond5eSheet#63]
Playing around with V10 to see what's compatible. I noticed that I'm getting an error when the module is looking for /systems/dnd5e/module/actor/sheets/character.js which doesn't exist anymore in DND5e 2.0.2.

Doesn't seem to be breaking anything, just worth mentioning. I saw that TidySheet and the full Beyond 5e Sheet are now updated, so the changes might be easily gleaned there. 

It looks like the file (not sure how much it's changed) now live in systems/dnd5e/templates/actors/character-sheet.hbs

**Comment by akrigline** [comment]
Thanks for investigating, but yeah definitely not v10 ready here yet.

**Comment by tehguitarist** [comment]
Yeah not a problem in the world, just thought I'd throw it out there. I'll play around and if it's a (very) simple fix I'll see if I can make a PR, but definitely a novice. No rush or worry at all.

**Comment by akrigline** [comment]
Haven't yet released, but will release a version with this fixed soon™
